saleor/graphql/campaign/enums.py

```diff
@@ -62,3 +62,133 @@ class EducationLeavelEnum(Enum):
     GCSE = "GCSE"
     MASTER = "MASTER"
     VOCATIONAL = "VOCATIONAL"
+
+
+class PeriodEnum(Enum):
+    MONTHLY = "MONTHLY"
+    YEARLY = "YEARLY"
+
+
+class EmploymentTypeEnum(Enum):
+    FIXED_TERM = "FIXED_TERM"
+    FIXED_TERM_WITH_PERMAMENT = "FIXED_TERM_WITH_PERMAMENT"
+    FREELANCE = "FREELANCE"
+    INTERSHIP = "INTERSHIP"
+    PERMAMENT = "PERMAMENT"
+    TRAINEESHIP = "TRAINEESHIP"
+
+
+class CurrenciesEnum(Enum):
+    AFN = "AFN"
```

Comment:
Can you confirm these are https://en.wikipedia.org/wiki/ISO_4217 codes?

Reply:
Quite sure there's several python libraries that bundle this enum.